content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/BtreeWatchCache.md

```diff
@@ -0,0 +1,20 @@
+---
+title: BtreeWatchCache
+content_type: feature_gate
+
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.32"
+---
+
+<!--
+When enabled, the API server will replace the legacy HashMap-based _watch cache_
+with a BTree-based implementation. This replacement may bring performance improvements.
+-->
+启用后，API 服务器将把传统的基于 HashMap 的**监视缓存（watch cache）**替换为基于 BTree 的实现。
```

Comment:
![image](https://github.com/user-attachments/assets/075114f9-c47e-4f3c-98dd-d674c66aa212)

There is a problem with the rendering of `**`